pkgs/applications/video/fake-background-webcam/default.nix

```
@@ -0,0 +1,51 @@
+{ lib
+, stdenvNoCC
+, fetchFromGitHub
+, python310
+}:
+
+stdenvNoCC.mkDerivation {
+  pname = "fake-background-webcam";
+  version = "unstable-2023-05-15";
```

```suggestion
  version = "0-unstable-2023-05-15";
```
https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md
> - The `version` attribute _must_ start with a digit e.g., `"0.3.1rc2"`.
>
> - If a package is a commit from a repository without a version assigned, then the `version` attribute _should_ be the latest upstream version preceding that commit, followed by `-unstable-` and the date of the (fetched) commit. The date _must_ be in `"YYYY-MM-DD"` format.
>
> Example: Given a project had its latest releases `2.2` in November 2021, and `3.0` in January 2022, a commit authored on March 15, 2022 for an upcoming bugfix release `2.2.1` would have `version = "2.2-unstable-2022-03-15"`.
>
> - If a project has no suitable preceding releases - e.g., no versions at all, or an incompatible versioning / tagging schema - then the latest upstream version in the above schema should be `0`.
>
> Example: Given a project that has no tags / released versions at all, or applies versionless tags like `latest` or `YYYY-MM-DD-Build`, a commit authored on March 15, 2022 would have `version = "0-unstable-2022-03-15"`.